packages/components/src/ItemList.tsx

```diff
@@ -181,26 +181,19 @@ export class ItemList<T> extends PureComponent<
     };
   }
 
-  componentDidMount(): void {
-    const { isStickyBottom } = this.props;
-    if (isStickyBottom && !this.isListAtBottom()) {
-      this.scrollToBottom();
-    }
-  }
-
   componentDidUpdate(
     prevProps: ItemListProps<T>,
     prevState: ItemListState
   ): void {
-    const { selectedRanges: propSelectedRanges } = this.props;
+    const { selectedRanges: propSelectedRanges, itemCount } = this.props;
     const {
       focusIndex,
       isStuckToBottom,
       scrollOffset,
       height,
       selectedRanges,
     } = this.state;
-    if (isStuckToBottom && !this.isListAtBottom()) {
+    if (isStuckToBottom && !this.isListAtBottom() && itemCount > 0) {
```

Comment:
I think we still need the `componentDidMount` - if you have a virtual list that's stuckToBottom and has items already loaded, it won't scroll to bottom.

Reply:
I could be wrong, I just tried this by modifying the styleguide and doesn't seem to be an issue. Must be something causing a re-render on mount

Reply:
I don't think it's needed, that's what the initialScrollOffset is for. when it mounts?